project-extender/src/providers/jira-cloud-provider/JiraCloudProvider.ts

```diff
@@ -200,15 +200,28 @@ class JiraCloudProvider implements ProviderApi {
         storyPointsEstimate: await this.getIssueStoryPointsEstimate(
           element.key
         ),
+        rank: element.fields[rankCustomField!],
         index,
       }))
     )
 
     return issues
   }
 
-  async moveIssueToSprint(sprint: number, issue: string): Promise<void> {
+  async moveIssueToSprintAndRank(
+    sprint: number,
+    issue: string,
+    rankBefore: string,
+    rankAfter: string
+  ): Promise<void> {
     return new Promise((resolve, reject) => {
+      const rankCustomField = this.customFields.get("Rank")
+      const body = {
+        rankCustomFieldId: rankCustomField!.match(/_(\d+)/)![1],
+        issues: [issue],
+        ...(rankAfter ? { rankAfterIssue: rankAfter } : {}),
```

Comment:
```suggestion
        ...(rankAfter ?? { rankAfterIssue: rankAfter }),
```
not sure check if it works

Reply:
![image](https://user-images.githubusercontent.com/77448275/216768178-8aa74de9-2de4-4d46-b2a1-dee9a7538609.png)